cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceMeta.java

```diff
@@ -63,6 +63,7 @@ public NamespaceConfig getConfig() {
     private String principal;
     private String groupName;
     private String keytabURI;
+    private Integer keytabURIVersion;
```

Comment:
Is this transparent to the user (such as if they use the REST API to see the namespace meta)?
Shouldn't it be transparent to the user?
@poornachandra

Reply:
Yes - the version should not be exposed to the user.

Reply:
Why is it `Integer`, not `int`?